src/solvers/dgsem_tree/container_viscous_2d.jl

```diff
@@ -39,7 +41,8 @@ end
 # `unsafe_wrap`ping multi-dimensional `Array`s around the
 # internal storage.
 function Base.resize!(viscous_container::ViscousContainer2D, equations, dg, cache)
-    capacity = nvariables(equations) * nnodes(dg) * nnodes(dg) * nelements(dg, cache)
+    capacity = nvariables(equations) * nnodes(dg) * nnodes(dg) *
+               nelements(dg, cache)
```

Comment:
Why this change? If it is to make the code more similar to the 3D case, you could use

```suggestion
    capacity = nvariables(equations) * nnodes(dg)^2 * nelements(dg, cache)
```

instead (and similar in 1D and 3D).